benchmark.py

```
@@ -0,0 +1,826 @@
+        if self.data_dir.exists():
+            shutil.rmtree(self.data_dir)
+        self.data_dir.mkdir(parents=True)
+        
+        command = [
+            self.config.server_exe,
+            '--dir', str(self.data_dir),
+            '--port', str(self.port),
+            '--log-level', 'debug',
+        ]
+        print(' '.join(command))
+        self.process = subprocess.Popen(
```

Ruff (0.12.2)

Async functions should not create subprocesses with blocking methods

(ASYNC220)